### PR TITLE
docs: Add docs for AKV backed KBS deployment

### PR DESCRIPTION
**Pull request by mkulke**

### CODE REVIEW

File: content/en/docs/key-broker-service/azure/kbs-backed-by-akv.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+- [Use the Azure Key Vault provider for Secrets Store CSI Driver in an Azure Kubernetes Service (AKS) cluster](https://learn.microsoft.com/en-us/azure/aks/csi-secrets-store-driver)
+- [Use Microsoft Entra Workload ID with Azure Kubernetes Service (AKS)](https://learn.microsoft.com/en-us/azure/aks/workload-identity-overview)
+
+### AKV
+
+There should be an AKV instance that has been configured with role based access control (RBAC), containing two secrets named `coco_one` `coco_two` for the purpose of the example. Find out how to configure your instance for RBAC in the guide below.
+
+[Provide access to Key Vault keys, certificates, and secrets with an Azure role-based access control](https://learn.microsoft.com/en-us/azure/key-vault/general/rbac-guid)
```
Comment (portersrc) [inline content/en/docs/key-broker-service/azure/kbs-backed-by-akv.md:27]: This 404s. Seems to be "rbac-guide" (with an "e") at the very end.

**Review by portersrc — APPROVED** [review]
This is awesome. Nice guide.

**Review by fitzthum — DISMISSED** [review]

File: content/en/docs/key-broker-service/azure/kbs-backed-by-akv.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+- [Use the Azure Key Vault provider for Secrets Store CSI Driver in an Azure Kubernetes Service (AKS) cluster](https://learn.microsoft.com/en-us/azure/aks/csi-secrets-store-driver)
+- [Use Microsoft Entra Workload ID with Azure Kubernetes Service (AKS)](https://learn.microsoft.com/en-us/azure/aks/workload-identity-overview)
+
+### AKV
+
+There should be an AKV instance that has been configured with role based access control (RBAC), containing two secrets named `coco_one` `coco_two` for the purpose of the example. Find out how to configure your instance for RBAC in the guide below.
+
+[Provide access to Key Vault keys, certificates, and secrets with an Azure role-based access control](https://learn.microsoft.com/en-us/azure/key-vault/general/rbac-guide)
+
+Note: you might have to toggle between Access Policy and RBAC modes to create your secrets on the CLI or via the Portal if your user doesn't have the necessary role assignments.
```
Comment (surajssd) [inline content/en/docs/key-broker-service/azure/kbs-backed-by-akv.md:29]:
```suggestion
> **Note**: You might have to toggle between Access Policy and RBAC modes to create your secrets on the CLI or via the Portal if your user doesn't have the necessary role assignments.
```

File: content/en/docs/key-broker-service/azure/kbs-backed-by-akv.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+EOF
+kubectl create -f secret-provider-class.yaml
+```
+
+### Deploy KBS
+
+The Secret Provider Class can be referenced in a KBS deployment. The secrets are mounted into the storage hierarchy `default/akv`.
+
+```bash
+export KBS_IMAGE=ghcr.io/mkulke/confidential-containers/kbs:7ec54fa8f74b9bf24d84ef4a35d050d068105732-nopolicy
```
Comment (surajssd) [inline content/en/docs/key-broker-service/azure/kbs-backed-by-akv.md:130]: What is the source of this image? How can users find about it?

**Review by surajssd — CHANGES_REQUESTED** [review]

**Comment by surajssd** [comment]
Eventually I can envision the docs section has following hierarchy

docs
- overview
- projects
  - kata
  - guest components
  - kbs
  - caa
- tutorials
  - kbs with akv
  - using encrypted container images with coco
  - ...
- concepts

File: content/en/docs/key-broker-service/azure/kbs-backed-by-akv.md (hunk 3 above)
Comment (mkulke) [reply content/en/docs/key-broker-service/azure/kbs-backed-by-akv.md:130]: Ah, it makes sense to kustomize the existing kbs deployments instead.

**Comment by surajssd** [comment]
Did you mean to create a sub-section called Azure under KBS docs? Because the URL and the breadcrumb on the docs don't match. There is no sub-section `azure` on the docs.

image

**Comment by mkulke** [comment]
> Did you mean to create a sub-section called Azure under KBS docs? Because the URL and the breadcrumb on the docs don't match. There is no sub-section `azure` on the docs. For this to happen we need one `_index.md` file there.

ah, no. I don't think that level of nesting is required. I moved it directly into kbs. eventually we might want to re-organize the hierarchy anyway.

**Review by surajssd — APPROVED** [review]

**Pull request merged**